examples/functions/newModes/NewModes-CarClubExample.xml

```diff
@@ -374,6 +379,7 @@
 							<members>
 								<Vehicle version="any" id="car_01">
 									<OperatorRef version="any" ref="coc:MCR"/>
+									<ContactRef ref="contact1" version="any"/>
 									<SimpleVehicleTypeRef version="any" ref="small_car"/>
 									<VehicleModelRef version="any" ref="mini_whiz"/>
 									<CarModelProfileRef version="any" ref="model_a"/>
```

Comment:
I'm puzzled because this still does not validated (error on  CarModelProfileRef ) as if the VehicleCodeGroup has changed position... but it does not seem to be the case !

Reply:
@Aurige  What is not validating?

Reply:
NeTEx\examples\functions\newModes\NewModes-CarClubExample.xml (and it validates in _master_, so we broke something at some point)

Reply:
@Aurige to provide the details of the error

Reply:
@Aurige for me it validates in the branch..
![image](https://github.com/NeTEx-CEN/NeTEx/assets/24227470/3d6e9bc8-9429-41ff-8e16-8b04cff3729a)

Reply:
The changes coming from this code are here. It is only Contact and ContactRef. However, we might have changed it before.

Reply:
I'm not sure what is going on, but when I checkout that brach and validate the CarClub example, I get this (I just double checked)
![image](https://github.com/NeTEx-CEN/NeTEx/assets/3328999/b187ee38-3e1d-4aa3-a0ec-b3803996a88a)

Reply:
Same pb with XMLLint

![image](https://github.com/NeTEx-CEN/NeTEx/assets/3328999/147a45d7-9ca8-491e-aef2-75b1eef17634)